canvas_modules/common-canvas/src/common-properties/components/title-editor/title-editor.jsx

```diff
@@ -136,19 +139,20 @@ class TitleEditor extends Component {
 				iconDescription={propertiesTitleEditButtonLabel}
 				hasIconOnly
 			/>);
-
 		const helpButton = this.props.help
-			? (<Button
-				kind="ghost"
-				className="properties-title-editor-btn help"
-				data-id="help"
-				onClick={this.helpClickHandler}
-				tooltipPosition="bottom"
-				renderIcon={Help}
-				size="sm"
-				iconDescription={helpButtonLabel}
-				hasIconOnly
-			/>)
+			? (<span className="properties-heading-icon">
+				<Tooltip
+					className="properties-title-editor-btn help"
+					data-id="help"
+					tip={this.props.help?.data?.description?.label}
+					link={this.props.help?.data?.description?.link ? this.props.help?.data?.description?.link : null}
+					tooltipLinkHandler={this.props.controller.getHandlers().tooltipLinkHandler}
+					direction="bottom"
```

Comment:
@nmgokhale here should `tooltipLinkHandler` be adjusted to call `helpClickHandler` ?